Lesson008/Lesson005/Task 2 HW05.IntuitionTraining/Program.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+
+namespace Task2HW05.IntuitionTraining
+{
+    internal class Program
+    {
+        static int HiddenNumber(int guess, int number, int min, int max)
+        {
+            while (guess != number)
+            {
+                Console.WriteLine("Guess a number between " + min + " - " + max + " : ");
+                guess = Convert.ToInt32(Console.ReadLine());
+                Console.WriteLine(" Guess: " + guess);
+
+                if (guess > number)
+                {
+                    Console.WriteLine("No number " + guess + " is too high!");
+                }
+                else if (guess < number)
+                {
+                    Console.WriteLine("No number " + guess + " is too low!");
+                }
+            }
+            Console.WriteLine("Number: " + number);
+            Console.WriteLine("YOU WIN!!!");
+            return number;
+        }
+        static void Main(string[] args)
+        {
+            Random random = new Random();
+            bool playAgain = true;
+            int min = 1;
+            int max = 5;
+            int number;
+            int guess;
+
+            while (playAgain)
+            {
+                guess = 0;
+                number = random.Next(min, max + 1);
+                int number1 = HiddenNumber(guess, number, min, max);
+            }
```

Comment:
if these variables do not have values, then it is not necessary to pass them to the method. it's illogical